hypha/apply/activity/tasks.py

```diff
@@ -8,6 +8,8 @@
 
 
 def send_mail(subject, message, from_address, recipients, logs=None):
+    if settings.EMAIL_SUBJECT_PREFIX:
+        subject = str(settings.EMAIL_SUBJECT_PREFIX) + str(subject)
```

Comment:
@sandeepsajan0 you think we need to add `' '` in b/w both strings?

Reply:
In Django the default value for this variable is

`EMAIL_SUBJECT_PREFIX = '[Django] '`

with a space at the end. I think best to stick with that for Hypha.

Reply:
@sks444 Yes, I also thought about adding `' '`, `-` or `:`  in b/w the strings but as Fredrik mentioned we already have space in its default value so we can expect a similar pattern.